### PR TITLE
feat 前端添加Slack机器人入口

### PR DESCRIPTION
**Pull request by stephenwang1011**
> Whether this PR is eventually merged or not, Sonic will thank you very much for your contribution. 
> 
> 无论此PR最终是否合并，Sonic组织都非常感谢您的贡献。

### Checklist

- [x] The title starts with fix, fea, or doc. | 标题为fix、feat或doc开头。
- [x] I have checked that there are no duplicate pull requests with this request. | 我已检查没有与此请求重复的拉取请求。
- [x] I have considered and confirmed that this submission is valuable to others. | 我已经考虑过，并确认这份呈件对其他人很有价值。
- [x] I accept that this submission may not be used. | 我接受此提交可能不会被使用。

### Description
需求来源：
https://sonic-cloud.wiki/d/2566-slack

### CODE REVIEW

File: src/components/ProjectUpdate.vue [hunk 1]
```diff
@@ -45,6 +45,7 @@ const robotList = [
   { name: '友空间机器人', value: 4, img: 'You' },
   { name: 'Telegram机器人', value: 5, img: 'Telegram' },
   { name: 'LINE Notify', value: 6, img: 'LineNotify' },
+  { name: 'Slack', value: 7, img: 'SlackNotify'},
```
Comment (ZhouYixun) [inline src/components/ProjectUpdate.vue:48]: 名字可以改为Slack Bot

File: src/views/Devices.vue [hunk 2]
```diff
@@ -134,6 +134,7 @@ const robotList = [
   { name: '友空间机器人', value: 4, img: 'You' },
   { name: 'Telegram机器人', value: 5, img: 'Telegram' },
   { name: 'LINE Notify', value: 6, img: 'LineNotify' },
+  { name: 'Slack', value: 7, img: 'SlackNotify'},
```
Comment (ZhouYixun) [inline src/views/Devices.vue:137]: 同上

**Comment by ZhouYixun** [comment]
pr规范仍需更改，参考 https://github.com/SonicCloudOrg/sonic-server/pull/313

**Comment by ZhouYixun** [comment]
http://upload.techweb.com.cn/s/640/s/640/2018/0822/1534906970549.jpg

logo可以用这个

**Comment by stephenwang1011** [comment]
> 

好的  以上的 我来改正

**Review by ZhouYixun — APPROVED** [review]
允许合入

**Pull request merged**